core/src/test/java/org/everit/json/schema/DefaultInOneOfTest.java

```diff
@@ -0,0 +1,47 @@
+package org.everit.json.schema;
+
+import org.everit.json.schema.loader.SchemaLoader;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.json.JSONTokener;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import static org.everit.json.schema.TestSupport.loadAsV6;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class DefaultInOneOfTest {
+
+    private static final ResourceLoader LOADER = ResourceLoader.DEFAULT;
+
+    @Test
+    public void defaultInFirstOneOf() {
+        JSONObject jsonSchema = LOADER.readObj("default-in-first-oneof-schema.json");
+        JSONObject jsonSpecification = LOADER.readObj("default-in-oneof.json");
+        Schema schema = SchemaLoader
+                .builder()
+                .useDefaults(true)
+                .schemaJson(jsonSchema)
+                .build()
+                .load()
+                .build();
+        assertDoesNotThrow(() -> schema.validate(jsonSpecification));
+    }
+
+    @Test
+    public void defaultInLastOneOf() {
+        JSONObject jsonSchema = LOADER.readObj("default-in-last-oneof-schema.json");
+        JSONObject jsonSpecification = LOADER.readObj("default-in-oneof.json");
+        Schema schema = SchemaLoader
+                .builder()
+                .useDefaults(true)
+                .schemaJson(jsonSchema)
+                .build()
+                .load()
+                .build();
+        assertDoesNotThrow(() -> schema.validate(jsonSpecification));
```

Comment:
Can you please verify in these 2 tests that the correct default value is picked up?

Reply:
Done